interpreter/cling/lib/Interpreter/ClingPragmas.cpp

```diff
@@ -33,26 +85,12 @@ namespace {
                       PragmaIntroducerKind Introducer,
                       Token &FirstToken) override {
       // TODO: use Diagnostics!
+      ParseResult_t Diagnostics = HandlePragmaHelper(PP, FirstToken, m_Interp, "pragma cling load");
```

Comment:
`Diagnostics` seems to not reflect that it can contain the string value, diagnostics are error or warning or info messages - that's not the case here. So maybe `Result`?